src/cascade/stats/estimation.py

```diff
@@ -18,6 +20,8 @@ def meas_bounds_to_stdev(df):
     Replace any zero values with :math:`10^{-9}`.
     """
     MATHLOG.debug("Assigning standard error from measured upper and lower.")
+    if np.any(df.meas_lower > df.meas_value) or np.any(df.meas_value > df.meas_upper):
+        raise ValueError(f"Found data where meas_lower > meas_value or meas_upper < meas_value")
```

Comment:
This would have helped earlier. If you have time, and sanity enough, it would be cool to report how many lines are out of whack. Not necessary.

Reply:
Yeah, I agree. I was trying to do the quick thing but I guess there isn't really a rush. I'll improve the message.